\documentclass[11pt,twocolumn]{article} 
\usepackage{natbib}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{hyperref}
\usepackage[margin=.5in]{geometry}

\graphicspath{{figures/}}

\title{Modeling How Evolution in One Trait is Affected by Adaptation in Another}
\date{Fall 2016}
\author{Kevin Gomez,Jason Bertram,Joanna Masel}

\begin{document}
\maketitle
\newpage

\begin{abstract}
Abstract.
\end{abstract}

\section{Introduction}

Natural selection frequently acts on multiple traits simultaneously. A single trait's response to selection leads to a shift in the population mean towards the fittest phenotype. With multiple traits the response involves both direct and indirect effects that together determine the evolution each trait \citep{lande1983measurement,Lovell2013,Wagner2011}. Direct effects are those induced by selection favoring fitter phenotypic values in each of the traits.  However, traits may be associated at the genetic level in a manner that permits only certain combinations of their phenotypic values.  As a result, selection on any one trait produces to changes in the others due to the constraints on their possible combinations. For example, we can consider two traits that are controlled by a single pleiotropic locus, but with only the first under selection. Allele frequencies at the locus change in response to selection against the first trait, and they also induce a shift in the mean of the second. These are indirect effects experienced by the second trait, which are caused by selection on the first.  Associations such as these can alter fundamental features of adaptation and have significant evolutionary consequences \citep{Felsenstein1979, Arnold2001, Arnold2008}.\par
% * <masel@email.arizona.edu> 2017-08-22T16:40:07.527Z:
% 
% > permits only certain combinations of their phenotypic values
% The rest of this paragraph seems to be about correlated traits, but here you talk about hard constraints. The two are not equivalent, indeed you can have either one without the other. I'm not totally sure what you want to achieve in this paragraph, other than giving a non-mathematical version of the next paragraph on Lande's equation. If so, I wonder if it is better to merge the two paragraphs, while removing the language about absolute constraints.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-22T20:17:12.979Z:
% 
% 
% 
% ^.
% * <masel@email.arizona.edu> 2017-07-28T22:53:26.215Z:
% 
% > trait expression,
% "Expression" is a funny word to use. For most biologists, it means gene expression. By calling it "trait expression", you are signalling that you mean something else, but I think readers might still find it confusing. The term "expression" pre-supposes that the genetic basis for something is there, and focuses on whether it is somehow realized or not; I don't think that is what you mean.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-22T14:05:36.066Z.
% * <masel@email.arizona.edu> 2017-07-28T22:52:47.005Z:
% 
% > the response involves both direct and indirect effects
% You need to define what you mean by direct and indirect effects: the reader doesn't know.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-07T14:10:25.416Z.

For quantitative traits, genetic associations and their effects on adaptation are mathematically described by Lande's equation ($\Delta \bar{z} = \textbf{G} \hspace{.05in}\beta$). It states that the change in the population mean $\Delta \bar{z}$ equals the product of the selection gradient $\beta$ and what is known as the \textbf{G}-matrix, whose components are the additive genetic covariances between traits. Lande’s result has been used to extend much of our understanding of how phenotypes evolve under selection, yielding a rich set of ideas that emphasize the role of the \textbf{G}-matrix in directing adaptive evolution at both micro- and macro-evolutionary scales.\par
% * <masel@email.arizona.edu> 2017-07-29T12:43:16.916Z:
% 
% > When combined with the notions of adaptive landscapes
% The link to adaptive landscapes is not obvious, including to me. G-matrix theory is all continuous, and the only possible landscape is a phenotype-fitness map. The adaptive landscape you are going to deal with is a genotype-fitness map, ie a completely different object.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-07T14:10:30.176Z.

The behavior of the \textbf{G}-matrix over time largely depends on what are the causes of the genetic correlations. Correlations between quantitative traits can arise for a number of reasons, but they often have a genetic basis \citep{Saltz2017}. They arise from either pleiotropy, linkage disequilibria, or are the product of trans acting elements; the first two that are thought to be the most significant factors. Historically, there have been two schools of thought what the principle causes of genetic correlations are. The first was put forth by the Edinburgh school, which held that pleiotropy was principle source, since linkage disequilibrium could rapidly decay by recombination \citep[Chapter~20]{fox2006evolutionary}. This belief was largely motivated by interests the relationships between parent-offspring phenotypes within randomly mating populations, and its implications for evolution. The alternative view came out of the Birmingham school, which largely experimented with inbred lines of plants. Geneticists in this group believed that linkage disequilibria was much more important; a fact that often reflected in their experiments. Presently, the Edinburgh view is more widely accepted, and as a result, most models for the evolution of the \textbf{G}-matrix assume genetic correlations are pleiotropic and proceed with examining how evolutionary processes influence them. We know far less about the about the behavior genetic correlations from linkage disequilibria over time, but this is not much of an issue for most complex organisms.\par
% * <masel@email.arizona.edu> 2017-08-22T16:50:23.214Z:
% 
% >  most models for the evolution of the \textbf{G}-matrix assume genetic correlations are pleiotropic and proceed with examining how evolutionary processes influence them
% An important exception to note is the "Bulmer effect", which I believe describes how selection can cause LD even in the absence of linkage, which then of course appears in G. This is distinct from our work where linkage to the background on which a mutation first appear is the cause of LD, albeit later shaped by selection. I believe there is some material on the Bulmer effect in Bruce Walsh's book. It is important to review it just enough to distinguish what you are doing from what Bulmer already did.
% 
% ^.
% * <masel@email.arizona.edu> 2017-08-22T16:47:43.713Z:
% 
% > the product of trans acting elements
% This is not self-explanatory, and insofar as I can guess what it means, it seems to me to be a special case of pleiotropy rather than a third distinct possibility.
% 
% ^.

In the case of asexuals, the Birmingham view is more appropriate, and determining how the genetic correlations due to LD change over time is critical to our understanding of adaptation. Linkage disequilibria persists for much longer periods of time in a asexuals, which can adversely affect adaptation. This fact has been used to argue for the evolutionary advantages of sex and recombination \citep{Barton2005,Otto2009}. Consequently, we can learn a substantial amount about the evolution of their traits by examining how genetic correlations from LD evolve over time and influence adaptation. Doing so would not only reveal a great deal about some of the oldest and most abundant forms of life on the planet, such as bacteria, but could prove useful to future evolutionary experiments.\par

Lande’s equation describes the phenotypic evolution of populations with multiple characters from generation to generation.  Whether one can use this framework to uncover the history of natural selection in populations over large evolutionary timescales largely depends on the forces that influence the evolution of the G-matrix.  Over the past few decades there has been a significant amount of effort to shed light on the matter of G-matrix stability.  The analysis of genetic models for the adaptive evolution of multiple phenotypic traits indicate some of the conditions needed for stability\citep{Turelli1988,Jones2003}.  These works exclude the potential influences of linkage disequilibrium on the dynamics of variances and covariances over time, and as a result, our understanding of the role of linkage disequilibria in G-matrix evolution remains unclear.  Nonetheless, the question is significant there is empirical evidence suggesting that in the G-matrix is often conserved in natural populations \citep{Arnold1999,Roff2000,Steppan2002}.  However, much of what we know about G-matrix stability corresponds the examining evolutionary forces on 

Our best models for adaptation in asexuals make use of what are known as traveling waves, and are based on population genetics. These models have established important relationships between the population's parameter and key features of evolution in asexuals. Among these is the traveling wave model developed by \citet{desai2007beneficial} for a population whose size $N$ is fixed in which only beneficial mutations occur at the rate $U$. They also make the key assumption that all mutations all provide the same selective advantage $s$, which allows them to show that the population's rate of adaptation is given by  $v(N,s,U) = 2s^2 \ln(N s)/\ln^2(s/U)$. This is an important result, because it directly provides the additive variance in fitness by simply applying Fisher's fundamental theorem ($v=\sigma^2$). If we regard the population as having one adaptive trait, then $\sigma^2$ is in fact the trait's additive variance and we obtain a quantitative genetics result as well.\par
% * <masel@email.arizona.edu> 2017-08-22T17:28:18.736Z:
% 
% > the model developed by \cite{desai2007beneficial}
% This is the place to briefly summarize the core assumptions of the model. You have already given the assumption of strict asexuality, the others are constant N, absence of deleterious mutations, and lack of variation in s among beneficial mutants. The latter (combined with asexuality) is the key innovation that makes the model work, because it relieves the need to track the identity of individual mutations.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-27T02:56:42.167Z:
% 
% I've added a statement about Desai and Fisher's assumptions following the citation, and I've also slightly changed the sentences after.
%
% ^.

Unfortunately, if we apply Desai and Fisher's result to the case of two adaptive traits, we still can still solve for the overall fitness variance $\sigma^2$ of the population, but nothing can be said about the variances and covariance of the two traits. This highlights the shortcomings of the traveling wave model in the case of multiple traits, for which we should expect Lande's equation to hold. As an example, consider a single adaptive trait evolving with parameters $N$, $s$ and $U$, then its variance and rate of adaption are $v(U;N,s)$ according to the formula given above. If the population acquires a second adaptive trait with identical parameters $s$ and $U$, then the new rate of adaptation is clearly $v(2U;N,s)$. By symmetry we know that rate of adaptation in each trait alone must be $0.5 \hspace{.02 in} v(2U;N,s)$, and furthermore, the rate of adaptation in trait one is now less with the second trait. Lande's result implies that trait one's rate of adaptation, when evolving with the second, must equal the sum of its variances and covariance ($v_1 =\sigma_1^2 +\sigma_{12}$). Clearly, adding the second trait produces a new covariance term that affects $v_1$, but it also induces a change in the variance of the first trait. What we do not know is the amount that each of these contributes to the observed reduction in trait one's rate of adaptation. This is exactly what we aim to determine. \par
% * <masel@email.arizona.edu> 2017-07-29T13:00:14.234Z:
% 
% > Clearly, we lack any information about how trait two is affecting trait one's evolution,
% Spell out what exactly it is that we don't know, ie the degree to which the reduction in adaptation rate of trait 1 is due to a change in variance or a change in covariance.
% 
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-22T14:11:45.747Z.

To explore how adaptation in one trait is affected by evolution in another, we extend the work of Desai and Fisher's traveling wave model and consider a two dimensional trait space. We examine the simplest case where individuals have two non-pleiotropic traits that contribute equally to fitness, each controlled by separate loci. We assume that beneficial mutations occur with identical rates in each trait, affect each independently, and exhibit no epistatic interactions.  Lastly, we will focus on a population whose size remains fixed. Our aim is to examine the effects of genetic correlations arising solely from linkage disequilibrium and determine how they influence trait evolution. 

\section{Materials and Methods}
\label{sec:materials:methods}

We considered an asexual population at carrying capacity $N$ in which haploid individuals had two traits contributing to their relative fitness. Each trait was assumed to be under the control of a single locus. We assumed that beneficial mutations improving each trait appeared at a rate of $U$ and increase th fitness by $s$; deleterious mutations were excluded. Individuals that had accumulated $i$ mutations in the first and $j$ in the second were taken to have an overall fitness of $r_{ij} = i s+j s$.  Their relative fitness is given by the difference $(r_{ij}-\bar{r})$, where  $\bar{r} $  is the population mean fitness.  We will often right $r_i$ in place of $i s$ and $r_j$ in place of $j s$, so that $r_{ij} = r_i + r_j$. 
% * <masel@email.arizona.edu> 2017-08-22T19:12:14.027Z:
% 
% > appeared at a rate of $U_k$ ($k=1,2$) and increase th fitness by $s_k$
% Since you are going to make U and s independent of k, it's easier on the reader if you do that from the outset, rather than set up the general case and then  collapse it.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-23T16:45:52.343Z:
% 
% I will change the notation the instances of U_k and s_k to just U and s here and in the rest of the document. 
%
% ^.

In an analogous approach to that of \citet{desai2007beneficial}, we divided our population into classes based on the number of beneficial mutations in each trait. Abundances are accordingly denoted $n_{ij}$ and their frequencies by $p_{ij}$. We assumed that $1/N \ll s \ll 1$ to ensure that the set of classes could be neatly separated into two groups based on whether their behavior was deterministic and stochastic. Following the convention of Desai and Fisher, we refer to the set of deterministically behaving classes as the bulk, while those that grow stochastically and have a positive selective advantage ($r_{ij}>\bar{r}$) will be referred to as the stochastic front. Unlike in the case of Desai and Fisher's  traveling wave whose stochastic front is a single point, our stochastic front is the boundary of the bulk's distribution above the line crossing the population's mean fitness. This is shown in figure 1, where classes in the bulk are shown in shades of blue, while those belonging to the stochastic front are shown in shades of red. One reason why the growth of these two groups is treated differently has to do with the fact that only beneficial mutations in the stochastic front are consequential to adaptation. In asexuals, a large percentage of beneficial mutations are wasted since most occur poor genetic backgrounds, and their lineages remain relatively insignificant size and as a result, they do not contribute to the population's adaptive evolution. These are the beneficial mutations that occur within the bulk. On the other hand, mutations that do manage to occur on good genetic backgrounds will eventually grow large enough in size to influence the population's mean relative fitness over time. These are exactly the beneficial mutations produced at the edge of the bulk that give rise to classes in the stochastic front, marked by arrows in figure 1. Thus, the bulk's distribution determines the position and behavior of the stochastic front, but in time, the latter also shapes how the former shifts forward (fix this!).\par
% * <masel@email.arizona.edu> 2017-08-22T19:17:50.152Z:
% 
% > those that grow stochastically and have a positive selective advantage ($r_{ij}>\bar{r}$) will be referred to as the stochastic front
% This is a good place to point a key difference between you and prior work, namely that your front is a curve not a point.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-23T17:03:48.195Z:
% 
% I added a bit more after this sentence, and changed the subsequent one.
% "Unlike in the case of Desai and Fisher's  traveling wave whose stochastic front is a single point, our stochastic front is the boundary of the bulk's distribution above the line crossing the population's mean fitness. This is shown in figure 1, where classes in the bulk are shown in shades of blue, while those belonging to the stochastic front are shown in shades of red. "
%
% ^.

% *****************************************************************************************************************
% ***********************************FIGURE 1*2D*Travelling*Wave***************************************************
% *****************************************************************************************************************
\begin{figure}
\includegraphics[width=1\linewidth]{fig1a.pdf}
\label{Figure 1.}
\caption{\footnotesize Evolutuion of a two dimensional traveling wave. The distribution depicted was sampled from simulations.  Each box represents a class composed of individuals with an identical number of beneficial mutations in each trait. Classes with solid coloring form part of the bulk, while those with patterned interiors represent the stochastic front. The red dot marks the mean number of beneficial mutations in each trait. The dashed red line represents a lines of constant fitness. Classes whose centers lie along this line have relative fitness equal to the population mean, but vary in the number of beneficial mutations that have accumulated in each trait. Relative fitness increases along the perpendicular direction. Classes . Beneficial mutations and random sampling produce negligible effects in the changes of their abundances. In asexuals, only beneficial mutations that occur on the fittest backgrounds produce lineages that may eventually fix. These are marked with arrows into the stochastic front. Other lineages are out-competed and remain insignificant in size with respect to bulk. As new classes, the bulk expands in the direction of increasing relative fitness, and classes below the red line shrink and eventually disappear. As a result, the two dimensional trait distribution continually shifts in the direction of increasing relative fitness over time. (Simulation parameters: $N=10^9$, $s=10^{-2}$, and $U=10^{-5}$)}
\end{figure}
% *****************************************************************************************************************
% *******************************END*FIGURE 1**********************************************************************
% *****************************************************************************************************************


The dynamics of the bulk was modeled by a system of ordinary differential equations, in which abundances changed in proportion to their selective advantage (\textbf{eqt. 1}). 
\begin{equation} \label{eq:1}
\dot{n}_{ij}(t) = (r_{ij}-\bar{r}) n_{ij}(t). 
\end{equation}
Abundances in the stochastic front were negligible in size relative to $N$ due to our parameter assumptions stated earlier.  Since the bulk represents the majority of population, we assumed that the frequencies of the bulk followed an analogous ODE.
\begin{equation} \label{eq:2}
\dot{p}_{ij}(t) = (r_{ij}-\bar{r}) p_{ij}(t) 
\end{equation}
These frequencies determine all of the quantitative genetic measures that are of interest to us.\par

Classes in the stochastic front intermittently transition from stochastic to deterministic behavior and become part of the bulk once they have grown sufficiently large. Such classes are said to have established. Prior to establishment, random fluctuations due to drift drive the dynamics of abundances. The behavior of these classes can be modeled as a branching process fed by incoming mutants generated in adjacent, less fit, classes within the bulk. 

Each mutant lineage that appears in the stochastic front either goes establishes or goes extinct. The probability of fixation measures the chances of the former outcome, and it depends on the selective advantage of the mutant. If its genotype is $(i,j)$, then its selective advantage is $s_{ij}=(r_{ij}-\bar{r})$. Standard mathematical results for branching processes can be used to solve for the lineage's probability of fixation which is given by equation 3.
% * <masel@email.arizona.edu> 2017-08-22T21:57:02.810Z:
% 
% > Standard mathematical results for branching processes can be used to solve for the lineage's probability of fixation which is given by equation 3
% Eq. 3 is not in a familiar form to me, and you don't give a citation. The result we discussed at your oral exams was pi = s/sigma^2 where sigma is the standard deviation of offspring number given a particular fitness/expectation. D&F for-convenience choice was sigma=1, so pi=s.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-23T17:43:47.591Z:
% 
% Their initial discussion of the branching process uses "s" to state the general results of mutant lineages establishing, but they later use "qs" for their analysis of establishments at the nose.  Equation 3 is taken from equation 16 in Desai and Fisher, but is also a general result found in Allen's book "intro to stoch proc" page 262.
%
% ^.
\begin{equation} \label{eq:3}
\pi_{fix} = \frac{s_{ij}}{ 1+s_{ij}}
\end{equation}
The lineage establishes when it has reached a size if roughly $1/s_{ij}$, and its subsequent growth is exponential. \cite{desai2007beneficial} expressed the abundances of these lineages as $n_{ij}(t) = \frac{1}{s_{ij}} \exp[{s_{ij}(t-\tau)}]$. The random variable $\tau$ is the time of establishment and it captures the cumulative effects of the stochastic fluctuations.  In their initial analysis, Desai and Fisher solve for the probability density function of $\tau$ for lineages destined to establish. In our model, we find it more useful to consider the related random variable $\nu = \frac{1}{s_{ij}}\exp[-\Delta r_{ij}\tau]$ representing an initial size that produces the deterministic trajectory observed after establishment. Using the expression given by Desai and Fisher for the PDF of $\tau$, we can arrive at the cumulative distribution of  $\nu$ (equation 5) by following the analysis of \cite{Uecker2011}. 
% * <masel@email.arizona.edu> 2017-08-22T22:14:21.200Z:
% 
% > useful to consider the related random variable $\nu = \frac{1}{s_{ij}}\exp[-\Delta r_{ij}\tau]$ representing an initial size that produces the deterministic trajectory observed after establishment
% If doing this was Uecker & Hermisson's idea, then this is the place to give them credit/citation.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-23T17:49:29.395Z:
% 
% I modified the last sentence, but this is probably not clear.  I'll have to go back to Uecker and Hermisson to come up with a more detailed explanation concerning their analysis.
% 
% Using the expression given by Desai and Fisher for the PDF of $\tau$, we can arrive at the cumulative distribution of  $\nu$ (equation 5) by following the analysis of \cite{Uecker2011}. 
% 
% ^.
\begin{equation} \label{eq:4}
P(\nu \leq \nu_0) = 1- e^{-\pi_{fix} \nu_0}
\end{equation}
Each class in the stochastic front may consist of many mutant lineages. However, it is the arrival time of the first mutant lineage destined to establish that determines which class will join the bulk next. Once the genotype of this mutant lineage is obtained, its initial size $\nu$ can be sampled. The class can then be regarded as belonging to the bulk at the determined arrival time with initial size $\nu$. \par
% * <masel@email.arizona.edu> 2017-08-22T22:18:30.247Z:
% 
% >  its initial size $\nu$ can be sampled
% I think this is the first time in the Methods that you talk about sampling. The rest could be read as a general mathematical description, rather than a methodology for simulation, so this could cause confusion. Right now you begin talking about numerical implementation two paragraphs down from here, and this reference to sampling doesn't have that context.
% 
% ^.
% * <masel@email.arizona.edu> 2017-07-29T16:52:06.887Z:
% 
% > This was obtained using the expression below derived by \citep{Uecker2011} [eqt. 16b].
% Do you use the full generality of Uecker & Hermisson, or are earlier expressions, eg as used in Desai & Fisher, sufficient for your work?
% 
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-04T23:43:18.217Z.
In order to investigate how of traits affect one another, we identified the contributions of each trait to the evolution of the population. The rate of adaptation can obtained by differentiating the population's mean relative fitness $\bar{r}=\sum_{ij} p_{ij} r_{ij}$, which is itself the sum of the two marginal means, $\bar{r_1}=\sum_{ij} p_{ij} is_1$ and $\bar{r_2}=\sum_{ij} p_{ij} j s_2$.  It follows that the total rate of adaptation is sum of rates of adaptation corresponding to each trait ($\dot{\bar{r}}=\dot{\bar{r_1}} +\dot{\bar{r_2}}$).  We can apply Lande's result to the two components $\dot{\bar{r_1}} $ and $\dot{\bar{r_2}} $ and rewrite them in terms of the trait variances $\sigma_1^2 = \sum_{ij} p_{ij} (r_i-\bar{r_1})^2$ and $\sigma_2^2 = \sum_{ij} p_{ij} (r_j-\bar{r_2})^2$, and their covariance $\sigma_{12} =\sum_{ij} p_{ij} (r_i-\bar{r_1})(r_j-\bar{r_2})$.  In doing so, we obtain the system expressed in equation 6.
% * <masel@email.arizona.edu> 2017-08-22T22:41:29.374Z:
% 
% >  equation 6
% Instead of hard-coding equation numbers, you can create auto-numbered links - ask Jason how. Right now the number doesn't match. You should also link this more clearly to the way it was presented in the introduction, ie you should explain that this is simply $\Delta \bar{z} = \textbf{G} \hspace{.05in}\beta$ in the case of two traits both contributing directly to fitness
% 
% ^.
% * <masel@email.arizona.edu> 2017-07-29T17:21:52.605Z:
% 
% >  We found that each of these rates could then be rewritten in terms of trait variances and covariance
% This is not something that "we found" or "obtained". It is simply a statement of Lande's equation for an appropriate (normalized) definition of what a trait is.  Also, remember to define the various sigma terms.
% 
% ^.
% * <masel@email.arizona.edu> 2017-07-29T17:18:22.431Z:
% 
% > arrived at the expression $\dot{\bar{r}}=\dot{\bar{r_1}} +\dot{\bar{r_2}}$
% I'm OK going straight to the expression. Your derivation of it is hard to follow, but I'm not convinced it is necessary.
% 
% ^.
\begin{equation}\label{eq:5}
\left[
\begin{array}{c}
\dot{\bar{r_1}} \\
\dot{\bar{r_2}} 
\end{array}
\right]
=
\left[
\begin{array}{cc}
\sigma_1^2 & \sigma_{12} \\
\sigma_{12} & \sigma_2^2 
\end{array}
\right]
\left[
\begin{array}{c}
1 \\
1 
\end{array}
\right]
\end{equation}
The components of the selection gradient and trait response have been normalized in the expression above. \par

Our numerical implementation consisted of two routines which ran iteratively for the duration of the simulation. Each iteration began with solving for the time-dependent abundances in the bulk governed by \eqref{eq:1} over a time interval of one-thousand generations. We used Mathematica's standard numerical ODE solver to obtain the solutions and used them to determine the appearance time of the next mutant lineage destined to establish. Having computed the timing of this event, we evaluated the solutions and accordingly updated the abundances of the bulk. Any classes with less than one individual were removed.  The establishing class was then added to the bulk with an appropriate initial size. Following this step, the next iteration began with solving for the abundances of the new bulk, followed by the subsequent steps outlined above.\par

The contributions of the stochastic front were incorporated into the adaptive evolution of the population by simulating each establishment event. We did this by identifying the first mutant lineage destined to establish among all mutants appearing in the stochastic front. Mutants awhich are parallel to the line shownwhich are parallel to the line shownppear according to a time-inhomogeneous Poisson process with rate functions given by the abundances of the adjacent classes from which they originated. Consequently, the arrival time $\tau_l$ of the $l^{th}$ mutant generated from the class $n_{ij}(t)$ having a mutation in trait $k$ can be used to form the random variable
% * <masel@email.arizona.edu> 2017-08-22T22:55:54.046Z:
% 
% > can be used to form the random variable
% This is confusing. The equation is for the cumulant for the inhomogeneous rate, the equation does not describe a random variable.
% 
% ^.
\[ 
\int_0^{\tau_l} U_k n_{ij}(t) dt ,
\] 
which has the distribution of $l$ i.i.d exponentially distributed summed random variables. By randomly sampling from the this distribution, we could then solve for each arrival time $\tau_l$.   We then computed the probability of fixation using \textbf{equation 5}.  The integral was numerically approximated up to an appropriate cutoff.  To determine if a mutation would establish, we sampled a uniform random variable and checked it against the calculated $\pi_{fix}$. The next class to establish was chosen accordingly and incorporated into the bulk with a random initial size generated with from \textbf{equation 4}.\par
% * <masel@email.arizona.edu> 2017-08-22T22:57:52.494Z:
% 
% >  \textbf{equation 5}
% Again, replace all these with auto-numbered links to specific equations, not to hard-numbered text.
% 
% ^.

Simulations were initialized with monoclonal populations at carrying capacity consisting of individuals having no beneficial mutations in either trait and relative fitness set zero. We constructed three parameter sets, each one varying either $N$, $s$ and $U$ while fixing the others. Each was varied across a range constructed from values reported by evolutionary experiments involving either E. coli or S. cerevisiae \citep{desai2007speed,Levy2015,Perfeito2007}. We ran three sets of simulations, each set constructed with one varying parameter among the three. The mean values for trait one's variance and covariance were recorded in proportion to the variance of trait one when evolving independently.  

We allowed the simulation to run for 5,000 generations to allow the population to achieve beneficial mutation-selection balance before collecting data. This ensured that the transient effects from the initializing would not distort the statistics collected for the distribution. Following the burn period of the simulation, we recorded the existing classes in the bulk and their respective abundances. The data was used to compute the marginal mean fitness and variances of each trait, as well as their covariance. From these quantities we also computed the instantaneous rate of adaptation (\textbf{eqtn. 5}), the mean rate of fitness increase in each trait and the overall rate of adaptation.

\section{Results}
Adaptation in a second trait 

In each of our simulations we observed a 40\% decrease in the rate of adaptation of a trait as a result adding a second. The decrease was consistent across three parameter sets which varied either $N$, $s$ and $U$. These results also agree with predictions that can be derived from Desai and Fisher's theoretical work ($v(N,s,U)$). However, our simulations show that upon adding the second trait, the net change in $v_1$ arises from an increase in the first trait's variance and the formation of negative covariance. The first results in an increase in $v_1$, as a direct consequence of equation 5. This gain is more than offset by the negative covariance created between the two traits, whose magnitude is always substantially larger than the variance increase.  Figure 2 presents the calculated scaled mean variance of trait one, and its scaled covariance with the second trait, as a function of each varying population parameter. The graph demonstrates that within these parameter ranges, the new rate of adaptation $v_1$ as a percentage of the prior in every case is nearly 60\% less due solely to the large amounts of negative covariance that builds up between the two traits.\par

% *****************************************************************************************************************
% ***********************************FIGURE 2*Scaled Var/Cov Changes & Trait Reduction*****************************
% *****************************************************************************************************************
\begin{figure}[h]
\begin{subfigure}[b]{1\linewidth}
\centering
\includegraphics[width=.48\linewidth]{fig5d.pdf}
\includegraphics[width=.48\linewidth]{fig5e.pdf}
\end{subfigure}
\begin{subfigure}[b]{1\linewidth}
\centering
\includegraphics[width=.48\linewidth]{fig5f.pdf}
\includegraphics[width=.48\linewidth]{fig5g.pdf}
\end{subfigure}
\label{fig}
\caption{\footnotesize Contributions to rate of adaptation from changes in trait variance and covariance. (a) $s$ varies with $N$ and $U$ being kept fixed ($N= 10^7$, $U=10^-5$). (b) $U$ varies with $N$ and $s$ being kept fixed ($N= 10^7$,$s=10^-2$). (c) $N$ varies with $s$ and $U$ being kept fixed ($s=10^-2$,$U=10^-5$).}
\end{figure}
% *****************************************************************************************************************
% ***********************************END*FIGURE*2******************************************************************
% *****************************************************************************************************************

The decreases shown in the scaled mean values of $v_1$ that are plotted in figure 2 are nearly independent of the population parameters picked within ranges characterizing asexual evolution. In all three parameter sets, the decrease nearly is always approximately 40\% of the trait's prior variance. The insensitivity follows from Desai and Fisher's expression for the rate of adaptation, which exhibits a logarithmic dependence on $N$, $s$ and $U$. This leads to a general expression that can be obtained for the expected reduction in a trait's rate of adaptation as a result of adding  $(n-1)$ traits.
\[
\% \Delta v_1 
= 1-\frac{1}{n}\frac{v(N,s,nU)}{v(N,s,U)}
\]
Of the three parameters, changes in $s$ affect the expected decrease in scaled $v_1$ most (fig 2d). The total number of additional traits also lead to larger and larger reductions in $v_1$. However, the extent of this effect is marginally less with each additional trait, but quite substantive with the first few. \par

In our simulations, we observed highly correlated behavior between the variance trait one and its covariance with the second.  Their fluctuations shown in figure 3 are nearly in sync throughout duration of the simulations. This behavior can be explained by the effects that covariance has on the distribution of individual traits.  As previously mentioned, negative covariance measure associations between fitter values of one trait with poorer one in the other.  Consequently, the larger the magnitude of the negative covariance, the more likely it is that we will observe extreme values in each trait, with each these associated with the opposite opposite versions in the other trait.   (This section still needs more work)\par

% *****************************************************************************************************************
% ***********************************FIGURE*3*COVARIANCE*TIME*SCALES*ARGUMENT*??***********************************
% *****************************************************************************************************************
\begin{figure}[h!]
\begin{subfigure}[b]{1\linewidth}
\label{a}
\caption{Two Dimensional Distribution \& Covariance Fluctations}
\includegraphics[width=0.48\linewidth]{fig3_N-10p09_c1-0d01_c2-0d01_U1-1x10pn5_U2-1x10pn5_exp1.pdf}
\includegraphics[width=0.48\linewidth]{fig9a.png}
\end{subfigure}
\begin{subfigure}[b]{1\linewidth}
\label{b}
\caption{Abundances Along Axis of Total Relative Fitness}
\includegraphics[width=1\linewidth]{fig9b.png}
\end{subfigure}
\label{Fig 5}
\caption{\footnotesize Evolution of the G-matrix over time. (a) Time dependent trajectories of trait one's variance and covariance are plotted with their mean values. Both are highly correlated in their behavior and fluctuate substantially over time. (b) Two dimensional trait distribution with two sets of classes identified. The first set, whose boxes are shaded in with patterned gray, represent the set of classes with 38 total beneficial mutations.  At time $t$, the population mean fitness lies on the line of constant fitness given by 38 total beneficial mutations; mean fitness is marked by the black dot. At time $t+tau_q$ the mean shifts to the next line of constant fitness associated with 39 total beneficial mutations. Here, $\tau_q \sim v^(-1)$ is the mean time between establishements described in \cite{desai2007beneficial}. (b) Depictions of the associated one dimensional traveling wave at times $t$ and $t+\tau_q$. Due to the Gaussian shape of the one dimensional traveling wave, the covariance between traits is set by proportions of classes within the peak. At time $t$ the group with 38 total beneficial mutations determine covariance, while at time $t+\tau_q$ it will be the group with 39 total beneficial mutations that sets covariance. In between these times the covariance is approximately a convex combination of its values at these two times.}
\end{figure}
% *****************************************************************************************************************
% ***********************************END*FIGURE*3******************************************************************
% *****************************************************************************************************************

Under beneficial mutation-selection balance, the time scale governing the dynamics of the G-matrix components is given by the mean time between establishements $\tau_q \sim 1/v(N,s,2U)$ described by \cite{desai2007beneficial}. Figure 3 shows the fluctuations in the variance of trait one, and its covariance with trait two.  The two are are highly correlated in their dynamics as a result of their adherence to equation 5. When the population is beneficial mutation-selection balance, the left hand side of this expression is approximately constant, and as a result, variance and covariance satisfy a linear relationship on average. This allows us to describe the dynamics of G by understanding the behavior of covariance over time, which is entirely driven by changes in the associated one dimensional fitness distribution depicted in figure 3c. The peak of the traveling wave represents the largest portion of the population. The frequencies of classes in this group, which are those whose fitness is equal to the population's mean fitness, to a large extent determine the covariance of the two dimensional distribution. After $\tau_q$ generations the mean fitness shift forward, and the peak of the one dimensional traveling wave will then coincide with next group of classes that lie on the next line of constant fitness, whose fitness subsequently equals the mean fitness, as shown in figure 3b. These major transitions in covariance are governed by the rate of adaptation. In between the times of these transitions, the covariance will simply be a convex combination of the two adjacent sets of classes that lie on the lines of constant fitness below and above the mean. Minor fluctuations in these dynamics are contributed by the residual portion of abundances not included in these two groups.\par

\section{Discussion}

An interesting consequence of the dynamics of covariance invovles the relationship between stochastic front and its role in setting the covariance of the two dimensional trait distribution. In between establishement events for the one dimensional traveling wave, classes that lie along the line of constant fitness passing through the fittest part of the stochastic front their relative size to one another. OneThis process is exactly  manner describe by \cite{Desai2013}. Thus changes in the value of covariance in fact follow the transitions of the peaks the one dimensional traveling wave shift. These major transitions are consequently governed by the rate of adaptation and determined at the stochastic front.   

\subsection{Comparison with Barton and Otto's Negative Linkage Disequilibria Results}
In our model, covariance between traits is arises entirely from linkage disequilibrium. We can assume that specific alleles at either locus correspond to the number of beneficial mutations that have accumulated in the corresponding trait.  It follows that the $p_{ij}$ are the gametic frequencies for the genome consisting of allele $i$ paired with allele $j$, and $D_{ij} = p_{ij}-p_{i\cdot} p_{\cdot j}$ is their coefficient of linkage disequilibrium ( $p_{i\cdot}$ and $p_{\cdot j}$ are marginal distributions). By applying \textbf{equation 2}, we can derive a mathematical relationship between covariance and linkage disequilibrium (need to cite chapter 5, Walsh and Lynch).
% * <masel@email.arizona.edu> 2017-07-29T17:34:47.794Z:
% 
% > specific alleles at either locus correspond to the number of beneficial mutations that have accumulated in the corresponding trait
% Multiple loci can contribute to the same trait. An allele cannot be a "number of beneficial mutations", because an allele must pertain to one locus. Stick to the word "genotype" not "allele", eg "genotype ij has i beneficial mutations in trait 1 and j beneficial mutations in trait 2 compared to the ancestral population". This means that you cannot calculate linkage disequilibrium as you do: linkage disequilibrium is generally defined as being between two particular beneficial mutations. What you calculated was simply a covariance. Because of this confusion, I got a bit lost here and so skipped Equation 6.
% 
% ^.
\begin{equation}
\sigma_{12}=\sum_{ij}D_{ij}[(i-\bar{i})s_1][(j-\bar{j})s_2]. 
\end{equation} 
As the population adapts, mutations and selection  increase and decrease the amounts LD which ultimately determine how traits interact in their evolution.\par

We examined the resulting dynamics of trait evolution using numerical simulations of our model using Mathematica. We focused on modeling the symmetric case, in which the two traits had identical mutation rate and selection coefficients. These, along with the population size, were chosen to match values reported in evolutionary experiments involving relevant micro-organisms (see Table 1). 


\bibliography{biblio}
\bibliographystyle{plainnat}

\end{document}