\documentclass[9pt,twocolumn,twoside]{gsajnl}
% Use the documentclass option 'lineno' to view line numbers
\usepackage{caption}
\usepackage{subcaption}
\usepackage[nameinlink,capitalise]{cleveref}
\newcommand{\G}{\textbf{G }}

\articletype{inv} % article type
% {inv} Investigation 
% {gs} Genomic Selection
% {goi} Genetics of Immunity 
% {gos} Genetics of Sex 
% {mp} Multiparental Populations

\title{Linkage disequilibrium can drive strong and unstable variances and covariances in fitness-associated traits}

\author[$\ast$,1]{Kevin Gomez}
\author[$\dagger$]{Jason Bertram}
\author[$\dagger$]{Joanna Masel}

\affil[$\ast$]{Graduate Interdisciplinary Program in Applied Mathematics, University of Arizona, and}
% * <masel@email.arizona.edu> 2017-10-24T23:03:12.117Z:
%
% ^.
\affil[$\dagger$]{Department of Evolution and Ecology, University of Arizona}

\keywords{Keyword; Keyword2; Keyword3; ...}

\runningtitle{GENETICS Journal Template on Overleaf} % For use in the footer 

%% For the footnote.
%% Give the last name of the first author if only one author;
% \runningauthor{FirstAuthorLastname}
%% last names of both authors if there are two authors;
% \runningauthor{FirstAuthorLastname and SecondAuthorLastname}
%% last name of the first author followed by et al, if more than two authors.
\runningauthor{Gomez \textit{et al.}}

\begin{abstract}
Abstract.
\end{abstract}

\setboolean{displaycopyright}{true}

\begin{document}

\maketitle
\thispagestyle{firststyle}
\marginmark
\firstpagefootnote
\correspondingauthoraffiliation{Please insert the affiliation correspondence address and email for the corresponding author. The corresponding author should be marked with a `1' in the author list, as shown in the example.}
\vspace{-11pt}%

\section{Introduction}

Natural selection acts on multiple traits simultaneously. The mean trait value in a population can change either because of direct selection on trait X, or because of selection on trait Y plus a genetic correlation between X and Y \citep{lande1979quantitative,lande1983measurement}. The role of genetic correlations in the response of quantitative traits to natural selection is described by Lande's equation ($\Delta \bar{z} = \G \hspace{.05in}\beta$), where $\Delta \bar{z}$ is the expected change in the population mean of a trait $\Delta \bar{z}$, $\beta$ is the selection gradient, and the components of the \G  matrix are the additive genetic covariances between traits. Lande's equation governs the short-term evolution of correlated quantitative traits under natural selection, but it may play a role in bridging the gap between micro- and macro-evolution \citep{Arnold2001,Arnold2008,Schluter1996,Schluter2000}.\par

When the \G matrix is stable, it can be used to extrapolate future trait evolution from historical patterns, and to infer past selection gradients. The requirement for a stable \G matrix has motivated empirical work to measure and compare \G in related species. At least portions of the \G matrix may be conserved among certain phylogenetic groups \citep{Arnold1999,Roff2000,Steppan2002}, but other studies also indicate that \G is unstable. ***

Theoretical quantitative genetic models have also been used to examine the conditions under which the stability of the \G matrix can be expected \citep{Turelli1988}.  \par
% * <masel@email.arizona.edu> 2017-10-06T23:52:45.315Z:
% 
% > other studies also indicate that \G is unstable
% Citations(s) needed for empirical work for instability of G and theoretical work either deriving or assuming stability.
% 
% ^.

The genetic correlations described by \G are due to some combination of pleiotropy and linkage disequilibrium \citep{Saltz2017}. Historically, pleiotropy was favored by the Edinburgh school, who argued that recombination would quickly eliminate linkage disequilibrium \citep[Chapter~20]{fox2006evolutionary}, particularly in the randomly mating animal populations that the Edinburgh school focused on. In contrast, quantitative geneticists in the alternative, Birmingham school largely worked on inbred lines of plants, where the effects of linkage disequilibrium were impossible to ignore. The Edinburgh view came to be widely accepted, and as a result, most models for the evolution of the \G  matrix assume that genetic correlations are pleiotropic and go on to examine how evolutionary processes influence them \citep{Turelli1988}. We know far less about the about the dynamics of genetic correlations arising from linkage disequilibrium.\par
% * <masel@email.arizona.edu> 2017-10-24T22:58:18.095Z:
% 
% > most models for the evolution of the \G  matrix assume that genetic correlations are pleiotropic and go on to examine how evolutionary processes influence them \citep{Turelli1988
% More citations needed
% 
% ^.

Biology today has come to recognize the dominance of relatively asexual microbes \cite{mcfall2013animals}, for whom the Birmingham emphasis on linkage disequilibrium as a source of genetic correlations cannot be ignored. Linkage disequilibria certainly persist long enough to affect adaptation in asexuals; this fact has, for example, been the basis of explanations for the evolutionary advantages of sex and recombination \citep{Barton2005,Otto2009}. \par
% * <masel@email.arizona.edu> 2017-10-06T22:23:25.138Z:
% 
% > Biology today has come to recognize the dominance of relatively asexual microbes. 
% Cite www.pnas.org/cgi/doi/10.1073/pnas.1218525110 and maybe others making complementary points.
% 
% ^ <kgomez81@math.arizona.edu> 2017-12-05T18:40:59.153Z:
% 
% I added the citation.
%
% ^.
Asexual evolution is normally approached using a population genetic rather than a quantitative genetic approach. In particular, traveling wave models have been successfully used to describe the rate of multi-locus adaptation in asexuals given clonal interference \citep{rouzine2003solitary,desai2007beneficial}.  The basic model has only three parameters: fixed population size $N$, beneficial mutation rate $U$, and per-beneficial-mutation selection coefficient $s$. Adaptation, i.e. the increase in mean population fitness, occurs at the rate given by Desai and Fisher \citep[Equation 41]{desai2007beneficial}, expressed as
% * <masel@email.arizona.edu> 2017-10-14T17:09:10.651Z:
% 
% > Adaptation, i.e. the increase in mean population fitness, occurs at rate
% Cite the specific equation number in Desai & Fisher. Also give the parameter value conditions for which this equation holds.
% 
% ^ <kgomez81@math.arizona.edu> 2017-12-06T20:47:24.968Z:
% 
% I added the equation number to the citation.
%
% ^.
\begin{equation}\label{eq:1}
v(U,N,s) = s^2\frac{2\log(N s)-\log(s/U)}{\log^2(s/U)}.
\end{equation}
The expression holds when the asexual population is adapting in what is defined by Desai and Fisher as the ``concurrent mutations regime.'' In a monoclonal population, selection will cause a beneficial mutation to spread until all individuals carry the new mutation, at which point, it is said to have fixed in the population. The time required for beneficial mutation to fix is called its fixation time. For asexuals adapting in the concurrent mutations regime, beneficial mutations appear in rapid succession relative to the fixation time. This produces many mutant lineages that vary in the numbers of beneficial mutations that have accumulated in their genome. Beneficial mutations that appear on less fit lineages are lost due to clonal interference. Increasing the number of beneficial mutations that enter the population increases the proportion of beneficial mutations that are wasted.  The logarithmic dependence of $v(U,N,s)$ on the parameters captures these effects. An asexual population is in the concurrent mutations regime when the parameters satisfy the inequality $NU \gtrsim 1/\ln(Ns)$.  \par

Equation \ref{eq:1}, combined with Fisher's fundamental theorem, also gives the additive variance in fitness $\sigma^2$. With only one adaptive trait,  $\sigma^2$ also gives that trait's additive variance. With two adaptive traits, Equation \ref{eq:1} still gives overall fitness variance $\sigma^2$, but nothing can be said about the variances and covariance of the two traits.\par 

Adaptation in one trait proceeds more slowly when there is clonal interference with mutations that are adaptive in a second trait. Quantitatively, when the two adaptive traits each have mutations with fitness benefit $s$ appearing at rate $U$, the overall adaptation rate is $v(2U,N,s)$, and so by symmetry, adaptation in each trait alone occurs at rate $0.5 \hspace{.02 in} v(2U,N,s)$. This is lower than the rate $v(U,N,s)$ when the other trait is not also evolving. The reduction in trait-specific adaptation rate must be distributed between the two terms of Lande's equation $v_1 =\sigma_1^2 +\sigma_{1,2}$. Here, we will calculate how the reduction in trait-specific adaptation partitions between the variance in that trait and its covariance with the other.\par

% * <masel@email.arizona.edu> 2017-07-29T13:00:14.234Z:
% 
% > Clearly, we lack any information about how trait two is affecting trait one's evolution,
% Spell out what exactly it is that we don't know, ie the degree to which the reduction in adaptation rate of trait 1 is due to a change in variance or a change in covariance.
% 
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-22T14:11:45.747Z.

To explore how adaptation in one trait is affected by evolution in another, we extend the traveling wave model approach to a two dimensional trait space. We assume that each of the two fitness-determining traits is controlled by different loci, i.e. there is no pleiotropy. With pleiotropic contributions to \G excluded by design , we use this model to examine the dynamics of genetic correlations arising solely from linkage disequilibrium. 

\section{Materials and Methods}
\label{sec:materials:methods}

We consider an asexual population of haploids with fixed total population size $N$. To keep the model simple, we assume that beneficial mutations occur with the same rate of $U$ mutations per birth per trait with fitness benefit $s$ in each trait and no epistasis. Deleterious mutations are not considered. Individuals with $i$ mutations in the first trait and $j$ in second, have relative fitness $r_{i,j}= i s+j s$ and absolute fitness $r_{i,j}-\bar{r}$, where $\bar{r}$ is the population mean fitness. We refer to each such set of individuals as a ``class'' and denote their abundances and frequencies by $n_{i,j}$ and $p_{i,j}$. \par 

We assume $1/N \ll s \ll 1$.  Following the convention of Desai and Fisher, we refer to the set of classes abundant enough to behave deterministically $(n_{i,j}>1/s)$ as the bulk, and to smaller adaptive classes ($r_{i,j}>\bar{r}$) as the stochastic front (Figure \ref{fig:1}, solid and patterned, respectively). The stochastic front shares the same role of the nose in Desai and Fisher's traveling wave model \cite{desai2007beneficial}. Like the nose, the stochastic front contains the beneficial mutations (arrows in Figure \ref{fig:1}) that will drive future adaptation in the bulk; mutations within the bulk are wasted. However, the front in our model is evolving boundary rather than a point. Classes in the front become part of the bulk once they have ``established'' (grown sufficiently large). This shifts the position of  the bulk, with the stochastic front shifting also, to form a two dimensional traveling wave. \par 
% * <masel@email.arizona.edu> 2017-08-22T19:17:50.152Z:
% 
% > those that grow stochastically and have a positive selective advantage ($r_{i,j}>\bar{r}$) will be referred to as the stochastic front
% This is a good place to point a key difference between you and prior work, namely that your front is a curve not a point.
% 
% ^ <kgomez81@math.arizona.edu> 2017-08-23T17:03:48.195Z:
% 
% I added a bit more after this sentence, and changed the subsequent one one more sentence to indicate the differences in between Desai and Fisher's traveling wave nose. "The stochastic front shares the same role of the nose in Desai and Fisher's traveling wave model\cite{desai2007beneficial}. Like the nose, the stochastic front contains the beneficial mutations (arrows in Figure \ref{fig:1}) that will drive future adaptation in the bulk; mutations within the bulk are wasted."
% 
% ^.

\begin{figure}[!ht]
\includegraphics[width=1\linewidth]{figures/fig1a.pdf}
\caption{Illustration of a two dimensional trait distribution taken from a simulation. Individuals having an equal number of beneficial mutations in each trait are grouped into classes (boxes). Classes growing deterministically form part of the bulk (solid interiors), and those growing stochastically belong to stochastic front (patterned interiors). The red dot marks the average number of beneficial mutations in each trait. Fitness-isoclines identify the set of equally fit classes and are parallel the population mean fitness isocline (dashed red line). In asexuals, beneficial mutations must occur on the fittest backgrounds in order to contribute to the adaptive process. These mutations are represented by arrows from the bulk into the stochastic front. Over time, classes in stochastic front become part of the bulk once their abundances have become sufficiently large, while new ones subsequently emerge and become part of the stochastic front. Classes in the bulk below the population mean fitness isocline dwindle in size by selection. The distribution shifts direction of increasing relative fitness as result of these two processes and produce a two-dimensional traveling wave. (Simulation parameters: $N=10^9$, $s=0.02$, and $U=10^{-5}$)}\label{fig:1}
% * <masel@email.arizona.edu> 2017-10-14T18:05:49.152Z:
% 
% > These groups are represented by the lines passing through their centers that are parallel to the dashed line.
% Do you want to put these lines into the figure, or take this sentence out of the figure legend?
% 
% 
% ^.
\end{figure}

Changes in abundances within the bulk follow a system of ordinary differential equations given in \cite[page 27]{Crow1970} :
\begin{equation} \label{eq:2}
\dot{n}_{i,j}(t) =\left (r_{i,j}-\frac{N(t)}{N}\bar{r}\right) n_{i,j}(t),
\end{equation}
where time $t$ is measured in generations. At capacity, Equation \ref{eq:2} gives the selective advantage of a class relative to population mean fitness, denoted $s_{i,j}$, whose value is approximately  $s_{i,j} \approx (r_{i,j}-\bar{r})$. The stochastic front contains so little of the total population that variances and covariance are effectively determined only by the frequencies of classes within the bulk of two dimensional distribution. Each iteration began with solving (using NDSolve in Mathematica) for the time-dependent abundances in the bulk governed by \ref{eq:2} over a time interval of one thousand generations.\par

Growth of classes in the front can be modeled by branching processes fed by mutants from the adjacent (and exponentially growing) classes in the bulk. These mutations are shown by black arrows in Figure \ref{fig:1}. We sample the next mutation at the front by sampling random variable $x$ from a standardized exponential and solving $ x=\int_0^{\tau} b_{i,j} n_{i,j}(t)U dt$ for $\tau$. The birth rate in our model is set to $b_{i,j}=1+s_{i,j}$ per individual per generation.    

We then sampled whether the mutation would establish, by comparing a uniform random variable to the probability of fixation from Equation 16 of Desai and Fisher:

\begin{equation} \label{eq:3}
\pi_{fix} = \frac{s_{i,j}}{ 1+s_{i,j}}.
\end{equation}
After establishment of a mutant lineage at abundance $1/s_{i,j}$, it is part of the bulk, and subsequent growth is deterministic and exponential. \par

We update our system based on the first mutation to appear that is destined for establishment. The timing of establishment events is also stochastic, with later deterministic growth described by $n_{i,j}(t) = \frac{1}{s_{i,j}} \exp[{s_{i,j}(t-\tau)}]$ with the random variable $\tau$ measuring the time of establishment. We consider the related random variable $\nu = \frac{1}{s_{i,j}}\exp[-s_{i,j}\tau]$ representing the initial size at the time of first mutation that would produce the deterministic trajectory observed after establishment. Using the expression given by Desai and Fisher for the PDF of $\tau$ \citep[Equation 12]{desai2007beneficial}, we can arrive at the cumulative distribution of $\nu$:
% * <masel@email.arizona.edu> 2017-10-23T22:46:00.814Z:
% 
% > Using the expression given by Desai and Fisher for the PDF of $\tau$
% Give an Equation number
% 
% ^ <kgomez81@math.arizona.edu> 2017-12-05T19:58:30.000Z:
% 
% Added the citation for the PDF of \tau needed to get the CDF of \nu, and removed the citation to Uecker and Hermisson previously included.
% 
% ^.

\begin{equation} \label{eq:4}
P(\nu \leq \nu_0) = 1- e^{-\pi_{fix} \nu_0}
\end{equation}and used it to set the initial abundance of  the establishing class equal to $\nu$. We update the other abundances of the bulk at this time. Any classes with less than one individual were removed. The next iteration began with solving for the timecourse of the new bulk, now including the newly established class, followed by the subsequent steps outlined above.\par

\begin{figure*}[!ht]
\centering
\includegraphics[width=1\linewidth]{figures/fig2.pdf} 
\caption{Measured averages of variance (open diamonds) and covariance (closed circles) for trait one obtained from simulations, along with the theoretically predicted drop in its rate of adaptation (solid line). In each graph, two population parameters are kept constant while the third is varied to determine its effects on the mean variance and covariance of trait one induced the by evolution with the second trait.  In every case, the variance of trait one increases ($\sigma_1^2 >1$), while trait interactions result in negative covariance whose magnitude is larger than the change in variance ($|\sigma_{1,2}| > \sigma_1^2-1$), indicating that the reduction in the rate of adaptation of trait one is the result of large negative covariances.  Mean variance and covariance are insensitive to changes in the population size $N$ (pane c), but exhibit some dependence on the values of $U$ and $s$ (panes a,b). The values  $s=0.02$, $U=10^{-5}$ and $N=10^9$ where used for parameters that were kept constant.}\label{fig:2}
\end{figure*}

Each class in the stochastic front may be a mixture of lineages, each arising from a different mutation event. However, we assume that the arrival time of the first mutant lineage destined to establish is the one determines which class will join the bulk next. With some probability, the  \par
% * <masel@email.arizona.edu> 2017-10-23T22:52:29.988Z:
% 
% > Each class in the stochastic front may be a mixture of lineages, each arising from a different mutation event. However, it is the arrival time of the first mutant lineage destined to establish that determines which class will join the bulk next. \par
% Requires discussion. Errors introduced when the first establishing is earlier than the establishing time of the first arrival destined to establish.
% 
% ^.

Simulations were initialized with monoclonal populations at carrying capacity consisting of individuals having no beneficial mutations in either trait and relative fitness set to zero. We varied either $N$, $s$ and $U$ while fixing the others. Each was varied across a range constructed from values reported by evolutionary experiments involving either E. coli or S. cerevisiae \citep{desai2007speed,Levy2015,Perfeito2007}.  \par
% * <masel@email.arizona.edu> 2017-10-23T22:58:03.572Z:
% 
% > We varied either $N$, $s$ and $U$ while fixing the others. Each was varied across a range constructed from values reported by evolutionary experiments involving either E. coli or S. cerevisiae
% More detail on what exactly we learned from each paper, and also how we kept within concurrent-mutations regime, including saying what the concurrent mutations regime, or do the latter around Equation 1, and refer to it here.
% 
% ^ <kgomez81@math.arizona.edu> 2017-12-06T20:49:15.515Z:
% 
% I added a small discussion of the concurrent mutations regime near equation 1.
%
% ^.

The evolution of the population can be broken down into the evolution of the two traits, according to Lande's equation:
\begin{equation}\label{eq:5}
\left[
\begin{array}{c}
\dot{\bar{r_1}} \\
\dot{\bar{r_2}} 
\end{array}
\right]
=
\left[
\begin{array}{cc}
\sigma_1^2 & \sigma_{1,2} \\
\sigma_{1,2} & \sigma_2^2 
\end{array}
\right]
\left[
\begin{array}{c}
1 \\
1 
\end{array}
\right]
\end{equation}where  $\bar{r_1}=\sum_{i,j} p_{i,j} is_1$, $\bar{r_2}=\sum_{i,j} p_{i,j} j s_2$, $\sigma_1^2 = \sum_{i,j} p_{i,j} (r_i-\bar{r_1})^2$, $\sigma_2^2 = \sum_{i,j} p_{i,j} (r_j-\bar{r_2})^2$, and  $\sigma_{1,2} =\sum_{i,j} p_{i,j} (r_i-\bar{r_1})(r_j-\bar{r_2})$. Lande's equation above is expressed as the change in the population's mean trait values over time intervals of size $s$. \par
% * <masel@email.arizona.edu> 2017-10-21T16:44:31.244Z:
% 
% > have been normalized
% How have they been normalized?
% 
% ^ <kgomez81@math.arizona.edu> 2017-12-05T20:31:15.198Z:
% 
% Sorry, its time that's scaled (not normalized) to Lande's expression above.
% 
% ^.


We allowed the simulation to run for 5,000 generations to allow the population to achieve beneficial mutation-selection balance before collecting data. This ensured that the transient effects from the initializing would not distort the statistics collected for the distribution. Following the burn-in period of the simulation, we recorded the existing classes in the bulk and their respective abundances. The data was used to compute the marginal mean fitness and variances of each trait, as well as their covariance. From these quantities we also computed the instantaneous rate of adaptation (Equation \ref{eq:5}) for each trait and overall.\par
% * <masel@email.arizona.edu> 2017-10-23T23:12:23.801Z:
% 
% >  5,000 generation
% Need to define/explain what a generation is in the model.
% 
% ^.

\section*{Results}
\label{sec:results}

The reduction in trait-specific adaptation $v_1$ from $v(U,N,s)$ down to $0.5 \hspace{.02 in} v(2U,N,s)$ (Figure \ref{fig:2}, solid lines), caused by clonal interference with the second trait, is driven by high levels of negative covariance (Figure \ref{fig:2}, closed circles). This is partly compensated for by a substantial increase in additive genetic variance (Figure \ref{fig:2}, open diamonds), because the negative covariance slows the removal of additive variance from the population.

In other words, clonal interference in a rapidly adapting population can explain the common observation of high genetic variance in a fitness-associated trait combined with strong negative covariance with other fitness-associated traits, even in the complete absence of pleiotropic trade-offs. This is striking, because this kind of evidence is often interpreted as evidence for trade-offs. \par

\begin{figure}[!ht]
\centering
\includegraphics[width=1\linewidth]{figures/fig3.pdf}
\caption{The rate of adaptation of a focal trait as a function of the number of traits undergoing adaptation. . decays to zero in the Equation \ref{eq:6}, but in practice the expression is only valid when $kU/s \ll 1$. The effects of clonal interference on trait evolution are most significant in the case of two traits. The values of the parameters are taken as $s=0.02$, $U=10^{-5}$ and $N=10^9$.}
% * <masel@email.arizona.edu> 2017-10-28T22:59:26.669Z:
% 
% > Each plot includes the measured values produced by simulations with either varying $s$, $U$, or $N$. 
% Aren't var and covar from simulations and v from the equation?
% 
% ^.
\label{fig:3}
\end{figure}

The reduction in $v_1$ (Figure \ref{fig:2}, solid lines) is insensitive to changes in the values of  $U$, $N$, or $s$, due to the effects of clonal interference. When there are $k$ traits with equal $U$ and $s$, the expected $v_1$ is: 
\begin{equation}\label{eq:6}
 \tilde{v}_1 (U,s,N;k) 
= \frac{1}{k}\frac{\log^2[s/U]\log[N^2s(kU)]}{\log^2[s/(kU)]\log[N^2sU]},
\end{equation}
in units of multiples of the rate of adaptation in isolation, $v(U,N,s)$. This expression is obtained from Equation \ref{eq:1} and plotted in Figure \ref{fig:3}. Each additional trait increases the total beneficial mutation rate by $U$, which intensifies the effects of clonal interference from trait interactions. The total rate of adaptation that is less than the sum of the individual rates of each trait in isolation. The shortfall is measured by the departure of $\tilde{v}_1$ from unity. \par

% * <masel@email.arizona.edu> 2017-10-28T23:18:33.323Z:
% 
% > decreases in $v_1$ from additional traits are larger with increasing $s$
% I don't see this clearly in the figure - are you sure this is worth commenting on?
% 
% ^.
% * <masel@email.arizona.edu> 2017-10-28T23:09:27.018Z:
% 
% > Changes in $s$ are the most influential at smaller effect sizes.
% Needs more explanation. Why do you say this? Are you just observing from Figure 2a? The more natural axis for Figure 2a, given Equation 6, would be a log-scale.
% 
% Isn't the more important point that s is more important than U or N, at least over the parameter range considered?
% 
% ^.
Under beneficial mutation-selection balance, the components of the \G matrix fluctuate significantly over time, far more than the instantaneous rate of adaptation, with transient spikes in additive genetic variance $\sigma_1^2$ mostly canceled out by transient spikes in negative covariance $\sigma_{1,2}$ (representative simulation shown in Figure \ref{fig:3}). As a result, fluctuations in variance and covariance can be substantially larger than those of total variance in fitness.\par

\begin{figure}[!ht]
\includegraphics[width=1\linewidth]{figures/fig4.pdf}
% * <masel@email.arizona.edu> 2017-10-28T23:28:47.376Z:
% 
% > fig3.pdf
% Need tick labels on the x-axis.
% 
% ^.
\caption{Trajectories of the variance of trait one (dashed line), covariance (dotted line), and the total variance in fitness of the population (solid line), plotted over 10,000 generations. The mean values of each are included as lines with matching line-styles. The \G matrix over this period of time is unstable. Its two components, trait one variance and covariance, fluctuate significantly relative to the variance in total fitness over identical time periods. The trajectories were sampled from one simulation with parameters $s=0.02$, $U=10^{-5}$ and $N=10^9$.}\label{fig:4}
% * <masel@email.arizona.edu> 2017-10-28T23:32:29.075Z:
% 
% > along with variance in total fitness
% I thought you were plotting sigma_1^2+sigma_12 which is the instantaneous adaptation rate of trait 1, but here you say that you are plotting something else. Please confirm.
% 
% ^.
\end{figure}

The two-dimensional traveling wave of Figure \ref{fig:1} can be projected onto a one-dimension traveling wave of fitness, where classes along a fitness isocline such as those along the red line in Figure \ref{fig:1}, are aggregated into fitness classes (Figure \ref{fig:4}). The shape of the one-dimensional fitness distribution is Gaussian and highly peaked near the population's mean fitness \citep{desai2007beneficial}. This means that the \G matrix will dominated by the distribution of genotypes in this fitness class (red line in Figure \ref{fig:1}, mode in (Figure \ref{fig:4}). As adaptation continues, a fitter class becomes the most abundant that will have a different distribution of genotypes and hence different trait variances and covariance. On average, the time required for this shift is the mean time between establishments at the nose of the of the traveling wave, denoted $\tau_q$ in \citet{desai2007beneficial}.

\begin{equation}\label{eq:8}
\tau_q = \frac{\log^2(s/U)}{s(2\log(Ns)-\log(s/U))}
\end{equation}\par

Genotypes frequencies within the peak of the traveling wave are set during the stochastic growth phase of the fitness class prior to establishing \citep{Desai2013}.  As previously discussed, beneficial mutations that give rise to mutants in the nose produce distinct but equally fit mutant lineages. Many of lineages become extinct, but those that manage to survive are subsequently and equally amplified by selection following establishment. Since the relative sizes of the surviving lineages with respect to one do not change once they begin growing deterministically, the frequencies of genotypes in a fitness class become ``frozen'' at the time of establishment. Beneficial mutations that occur after that time contribute very little to the size and composition of fitness class, since their corresponding lineages will be exponentially smaller than those which initially appeared. After a fitness class has established, it eventually goes on to become the dominant group of the fitness distribution, or rather, it forms the peak of the traveling wave. The time between its establishment and its succession as the dominant fitness class is called the sweep time (denoted $\tau_{sw}$). Consequently, the frequencies of genotypes in the nose determine the values of variance and covariances approximately $\tau_{sw}$ generations later. Figure \ref{fig:6} plots the correlation between trait covariances of the bulk and nose, as a function of a time offset. The signal peeks at approximately $\bar{\tau}_{fix}$ (mean time to fixation). It shows that covariances between traits are largely set by the behavior at the nose of the one dimensional traveling wave, which consists of genotypes along the lead fitness isocline through the stochastic front. 

\begin{figure}[!ht]
\includegraphics[width=0.49\linewidth]{figures/fig5a.pdf}
\includegraphics[width=0.49\linewidth]{figures/fig5b.pdf}
\caption{\footnotesize The dynamics of the \G matrix components are governed by time scale $\tau_q$ of the one-dimensional traveling wave of fitness classes. Variances and covariance are dominated by the set of genotypes that make up the peak of the one-dimensional traveling wave, whose shape is approximately Gaussian. At time $t$ the peak is composed of classes shown in blue, whose frequencies determine the components of the \G  matrix. At time $t+\tau_q$ the mean shifts (relatively abruptly, due to the exponential growth of classes fitter than the mean), and the set of genotypes shown in green become the dominant group, whose distribution dominates the new trait variances and covariance.}\label{fig:5}
% * <masel@email.arizona.edu> 2017-10-28T23:57:21.626Z:
% 
% > whose shape is approximately Gaussian
% The text hasn't really explained or provided a citation for this point.
% 
% ^.
\end{figure}

\begin{figure}[!ht]
\includegraphics[width=1\linewidth]{figures/fig6a.pdf}
\caption{Correlation between covariance of stochastic front and bulk.  The correlation between the two peaks when their signals are offset by the the mean time to fixation $\bar{\tau}_{fix}$ (marked by the blue line), indicating that the dynamics of the stochastic front explains 69\% of the covariance lated detected in the bulk $\bar{\tau}_{fix}$ generation after.}\label{fig:6}
% * <masel@email.arizona.edu> 2017-10-28T23:57:21.626Z:
% 
% > whose shape is approximately Gaussian
% The text hasn't really explained or provided a citation for this point.
% 
% ^.
\end{figure}

Because mutational influx depends on the abundances of adjacent less-fit genotypes, the values of variances and covariance will be correlated from one transition to the next. However, these correlations vanish after a period of time roughly equal to the time required for classes in the stochastic front become the dominant group. This time period can be expressed as the product $q \tau_q$, where $q = 2\log(Ns)/\log(s/U)$ is the lead of the traveling wave nose over its mean. Over these periods of time $q \tau_q$, fluctuations in covariance can be significantly larger than those occurring over periods equal to $\tau_q$, resulting in an unstable \G-matrix over the time scale of adaptation. \par
% * <masel@email.arizona.edu> 2017-10-29T00:04:55.206Z:
% 
% > these correlations vanish after a period of time roughly equal to the time required for classes in the stochastic front become the dominant group
% Why is this?
% 
% ^.

\section*{Discussion}
\label{sec:discussion}

%An interesting consequence of the dynamics of covariance involves the relationship between stochastic front and its role in setting the covariance of the two dimensional trait distribution. In between establishement events for the one dimensional traveling wave, classes that lie along the line of constant fitness passing through the fittest part of the stochastic front their relative size to one another. OneThis process is exactly  manner describe by \cite{Desai2013}. Thus changes in the value of covariance in fact follow the transitions of the peaks the one dimensional traveling wave shift. These major transitions are consequently governed by the rate of adaptation and determined at the stochastic front.   
%
%\subsection*{Comparison with Barton and Otto's Negative Linkage Disequilibria Results}
%In our model, covariance between traits is arises entirely from linkage disequilibrium. We can assume that specific alleles at either locus correspond to the number of beneficial mutations that have accumulated in the corresponding trait.  It follows that the $p_{i,j}$ are the gametic frequencies for the genome consisting of allele $i$ paired with allele $j$, and $D_{i,j} = p_{i,j}-p_{i\cdot} p_{\cdot j}$ is their coefficient of linkage disequilibrium ( $p_{i\cdot}$ and $p_{\cdot j}$ are marginal distributions). By applying \textbf{equation 2}, we can derive a mathematical relationship between covariance and linkage disequilibrium (need to cite chapter 5, Walsh and Lynch).
%% * <masel@email.arizona.edu> 2017-07-29T17:34:47.794Z:
%% 
%% > specific alleles at either locus correspond to the number of beneficial mutations that have accumulated in the corresponding trait
%% Multiple loci can contribute to the same trait. An allele cannot be a "number of beneficial mutations", because an allele must pertain to one locus. Stick to the word "genotype" not "allele", eg "genotype i,j has i beneficial mutations in trait 1 and j beneficial mutations in trait 2 compared to the ancestral population". This means that you cannot calculate linkage disequilibrium as you do: linkage disequilibrium is generally defined as being between two particular beneficial mutations. What you calculated was simply a covariance. Because of this confusion, I got a bit lost here and so skipped Equation 6.
%% 
%% ^.
%\begin{equation}
%\sigma_{1,2}=\sum_{i,j}D_{i,j}[(i-\bar{i})s_1][(j-\bar{j})s_2]. 
%\end{equation} 
%As the population adaptsAcknowledgments, mutations and selection  increase and decrease the amounts LD which ultimately determine how traits interact in their evolution.\par
%
%We examined the resulting dynamics of trait evolution using numerical simulations of our model using Mathematica. We focused on modeling the symmetric case, in which the two traits had identical mutation rate and selection coefficients. These, along with the population size, were chosen to match values reported in evolutionary experiments involving relevant micro-organisms (see Table 1). 

\section*{Acknowledgments}
\label{sec:acknowledgments}
We thank Patrick Phillips for helpful discussions. Funding was provided by the National Science Foundation (DEB-1348262) and the National Institutes of Health (T32 GM084905).
\bibliography{biblio}

\section*{Appendix}
The derivation of the expression given in \ref{eq:4} from Lande's equation is as follows. The derivative measuring the change in the mean of a trait $\dot{\bar{r_1}} \approx \Delta \bar{r_1}/\Delta t$ for a sufficiently small interval of time. Lande's original expression yields,
\[
\left[
\begin{array}{c}
\Delta \bar{r_1} \\
\Delta \bar{r_2} 
\end{array}
\right]
=
\left[
\begin{array}{cc}
\sigma_1^2 & \sigma_{1,2} \\
\sigma_{1,2} & \sigma_2^2 
\end{array}
\right]
\left[
\begin{array}{c}
\beta_1 \\
\beta_2 
\end{array}
\right]
\]
The selection gradient on the right hand side is the change in fitness with respect to change in the trait value, indicating that $\beta_1=\beta_2=s$. Dividing both sides by $\Delta t$ allows us to equate the left side to the column vector $(\dot{\bar{r_1}},\dot{\bar{r_2}})^T$, while the scaled selection gradient becomes $(s/\Delta t,s/\Delta t)$. Setting $\Delta t = s$ provides equation 5.   

\end{document}Acknowledgments